lib/internal/streams/from.js

```diff
@@ -76,12 +77,14 @@ function from(Readable, iterable, opts) {
       const resolved = await value;
       if (done) {
         readable.push(null);
-      } else if (readable.destroyed) {
-        await close();
-      } else if (readable.push(resolved)) {
-        next();
       } else {
-        reading = false;
+        const res = await value;
+        if (res == null) throw new ERR_STREAM_NULL_VALUES();
```

Comment:
I'm not sure I agree with this throwing. There is no way to catch this? I think it should do `readable.destroy(new ERR_STREAM_NULL_VALUES())`. @lpinca Thoughts?

Reply:
+1!